### PR TITLE
Adds new calculated field ATCUD value to PTGenericInvoice entity

### PR DESCRIPTION
**Pull request by rlapao**
Adds new calculated field ATCUD value to PTGenericInvoice entity and set on issue().

### CODE REVIEW

File: billy-portugal/src/main/java/com/premiumminds/billy/portugal/services/documents/PTGenericInvoiceIssuingHandler.java [hunk 1]
```diff
@@ -103,6 +103,15 @@ protected void validateDocumentType(TYPE documentType, TYPE expectedType, String
         String sourceHash = GenerateHash.generateSourceHash(invoiceDate, systemDate, formattedNumber,
                 document.getAmountWithTax(), previousHash);
 
+		final String atcud = invoiceSeriesEntity
```
Comment (froque) [inline billy-portugal/src/main/java/com/premiumminds/billy/portugal/services/documents/PTGenericInvoiceIssuingHandler.java:106]: please keep the style consistent. check other files as well.

File: billy-portugal/src/main/java/com/premiumminds/billy/portugal/services/export/saftpt/v1_04_01/PTSAFTFileGenerator.java [hunk 2]
```diff
@@ -781,24 +781,11 @@ private Invoice generateSAFTInvoice(PTGenericInvoiceEntity document)
 		InvalidInvoiceTypeException {
 		Invoice saftInv = new Invoice();
 
-		InvoiceSeriesEntity invoiceSeries =
-			this.daoInvoiceSeries.getSeries(
-				document.getSeries(),
-				document.getBusiness().getUID().toString(),
-				LockModeType.NONE);
```
Comment (froque) [inline billy-portugal/src/main/java/com/premiumminds/billy/portugal/services/export/saftpt/v1_04_01/PTSAFTFileGenerator.java:788]: remove unused LockModeType import

File: billy-portugal/src/main/java/com/premiumminds/billy/portugal/services/export/saftpt/v1_04_01/PTSAFTFileGenerator.java [hunk 3]
```diff
@@ -781,24 +781,11 @@ private Invoice generateSAFTInvoice(PTGenericInvoiceEntity document)
 		InvalidInvoiceTypeException {
 		Invoice saftInv = new Invoice();
 
-		InvoiceSeriesEntity invoiceSeries =
-			this.daoInvoiceSeries.getSeries(
```
Comment (froque) [inline billy-portugal/src/main/java/com/premiumminds/billy/portugal/services/export/saftpt/v1_04_01/PTSAFTFileGenerator.java:785]: remove unused DAO

**Review by trcdl — COMMENTED** [review]
There seems to be a styling problem (your editor is probably configured to enforce tabs and this project uses spaces historically).

**Review by trcdl — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -20,30 +21,30 @@
...
  - [Removes hardcoded temporary file for SAFT export](https://github.com/premium-minds/billy/pull/124) (fixes [Concurrency bug in saft generation](https://github.com/premium-minds/billy/issues/7))
  - [Bump tiles-maven-plugin from 2.18 to 2.19](https://github.com/premium-minds/billy/pull/139)
- 
+
 ### Fixed
 
 - [Corrects the hash verification field](https://github.com/premium-minds/billy/pull/126)
 - [Adds missing Credit or Debit information to some invoice types](https://github.com/premium-minds/billy/pull/138). SQL script to fix previous invoices:
   ```
   UPDATE billy_core_generic_invoice SET credit_or_debit = 'CREDIT'
```
Comment (trcdl) [inline CHANGELOG.md:30]: Just remembered this now. Why not include the schema migration here as well?

**Review by trcdl — APPROVED** [review]

**Pull request merged**